securedrop/source_app/forms.py

```diff
@@ -5,17 +5,21 @@
 from wtforms import FileField, PasswordField, TextAreaField
 from wtforms.validators import InputRequired, Regexp, Length, ValidationError
 
-from models import Source, Submission
+from models import Submission
 
 
 class LoginForm(FlaskForm):
+
+    # Don't bother checking excessively long codenames to prevent DoS
+    _MAX_CODENAME_LEN = 128
```

Comment:
I think this should be defined where the passphrases are generated. The `Source` model is probably not the right place any more, but the `PassphrasesGenerator` could make its parameters public.

Reply:
Ok I can make the change